planemo/galaxy/config.py

```diff
@@ -1240,6 +1254,7 @@ def _install_with_command(ctx, config_directory, command, env, kwds):
     else:
         pip_install_command = ""
```

Comment:
I just noticed that, since commit 9867e5644a5f3a8f9d58c6896ad34df936bb4e48, `pip_installs` and `pip_install_command` could be removed, if desired.